tests/e2e/e2e_suite_test.go

```diff
@@ -18,226 +18,206 @@ package e2e_test
 
 import (
 	"bytes"
+	"context"
 	"fmt"
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
-	kubectl "github.com/rancher-sandbox/ele-testhelpers/kubectl"
-	"github.com/rancher/elemental-operator/tests/catalog"
 	"io"
 	"io/ioutil"
 	"net/http"
 	"os"
-	"strconv"
 	"strings"
 	"testing"
 	"time"
-)
 
-var (
-	chart      string
-	externalIP string
-	magicDNS   string
-	bridgeIP   string
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	kubectl "github.com/rancher-sandbox/ele-testhelpers/kubectl"
+	elementalv1 "github.com/rancher/elemental-operator/pkg/apis/elemental.cattle.io/v1beta1"
+	e2eConfig "github.com/rancher/elemental-operator/tests/e2e/config"
+	managementv3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+	runtimeconfig "sigs.k8s.io/controller-runtime/pkg/client/config"
 )
 
-const operatorNamespace = "cattle-elemental-system"
-const operatorName = "elemental-operator"
+func init() {
+	utilruntime.Must(clientgoscheme.AddToScheme(clientgoscheme.Scheme))
+	utilruntime.Must(managementv3.AddToScheme(clientgoscheme.Scheme))
+	utilruntime.Must(elementalv1.AddToScheme(clientgoscheme.Scheme))
+	utilruntime.Must(apiextensionsv1.AddToScheme(clientgoscheme.Scheme))
+}
+
+const (
+	operatorNamespace         = "cattle-elemental-system"
+	operatorName              = "elemental-operator"
+	nginxNamespace            = "ingress-nginx"
+	nginxName                 = "ingress-nginx-controller"
+	certManagerNamespace      = "cert-manager"
+	certManagerName           = "cert-manager"
+	certManagerCAInjectorName = "cert-manager-cainjector"
+	rancherNamespace          = "cattle-system"
```

Comment:
this should be `cattleSystemNamespace` Rancher is creating and using many different namespaces